Source/Core/VRTheWorldTerrainProvider.js

```diff
@@ -86,8 +85,17 @@ function VRTheWorldTerrainProvider(options) {
     if (srs === "EPSG:4326") {
       that._tilingScheme = new GeographicTilingScheme({ ellipsoid: ellipsoid });
     } else {
-      metadataFailure(`SRS ${srs} is not supported.`);
```

Comment:
I think this path was not previously retried for a reason-- This is not a failed request like the others. This is a case of the result not being supported, which would not change upon retrying.

Reply:
Good catch. Fixed!